easybuild/easyconfigs/i/impi/impi-2019.8.254-iccifort-2020.3.275.eb

```
@@ -0,0 +1,40 @@
+# This is an easyconfig file for EasyBuild, see http://easybuilders.github.io/easybuild
+
+name = 'impi'
+version = '2019.8.254'
+
+homepage = 'https://software.intel.com/en-us/intel-mpi-library/'
+description = "Intel MPI Library, compatible with MPICH ABI"
+
+toolchain = {'name': 'iccifort', 'version': '2020.3.275'}
+
+source_urls = ['https://registrationcenter-download.intel.com/akdlm/irc_nas/tec/16814/']
+sources = ['l_mpi_%(version)s.tgz']
+checksums = ['fa163b4b79bd1b7509980c3e7ad81b354fc281a92f9cf2469bf4d323899567c0']
+
```

If UCX is no longer necessary, then we wil have to reintroduce `libibverbs` as a system dependency

```suggestion

osdependencies = [('libibverbs-dev', 'libibverbs-devel', 'rdma-core-devel')]

```

---

UCX is still a dependency (https://github.com/easybuilders/easybuild-easyconfigs/pull/11337#discussion_r496134534), we can forget about this.